tendrl-notifier.spec

```diff
@@ -0,0 +1,66 @@
+Name: tendrl-notifier
+Version: 1.4.2
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Tendrl Notifier
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/Notifier
+
+BuildRequires: python-gevent
+BuildRequires: pytest
+BuildRequires: systemd
+BuildRequires: python-mock
+
+Requires: tendrl-commons
+
+%description
+Python module for Tendrl Notifier
+
+%prep
+%setup
+
+# Remove bundled egg-info
+rm -rf %{name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -m  0755  --directory $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/notifier
+install -m  0755  --directory $RPM_BUILD_ROOT%{_datadir}/tendrl/notifier
+install -Dm 0644 tendrl-notifier.service $RPM_BUILD_ROOT%{_unitdir}/tendrl-notifier.service
+install -Dm 0644 etc/tendrl/notifier.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/notifier/notifier.conf.yaml
+install -Dm 0644 etc/tendrl/notifier_logging.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/notifier/notifier_logging.yaml
+install -Dm 0644 etc/tendrl/email.conf.yaml.sample $RPM_BUILD_ROOT%{_sysconfdir}/tendrl/notifier/email.conf.yaml
```

Comment:
@nthomas-redhat @brainfunked 
For now, we are going with same approach of having smtp configuration on a file on disk right?

Reply:
yes, it comes from file only.  from email.conf.yaml file